hyperspy/_signals/image_simulation.py

```diff
@@ -17,9 +17,9 @@
 # along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
 
 
-from hyperspy._signals.image import Image
+from hyperspy._signals.signal2D import Signal2D
 from hyperspy._signals.simulation import Simulation
 
 
-class ImageSimulation(Simulation, Image):
+class ImageSimulation(Simulation, Signal2D):
```

Comment:
I think that this one didn't require any modification since we have too keep the API intact.